data/mwaa/requirements/requirements.txt

```diff
@@ -0,0 +1,22 @@
+--constraint https://raw.githubusercontent.com/awslabs/aiops-modules/main/data/mwaa/requirements/constraint-requirements.txt
```

Comment:
I'm not familiar with how constraints work but are we unable to reference the local file? I'm just worried that if we make a change to this requirements file and the constraints file in a dev branch, it will still be pulling the constraints file from `main`.